gcc/config/riscv/t-zephyr

```diff
@@ -76,6 +77,7 @@ march=rv64imafd_zicsr_zifencei/mabi=lp64d/mcmodel=medany
 MULTILIB_REUSE = \
 march.rv32i_zicsr_zifencei/mabi.ilp32=march.rv32ia_zicsr_zifencei/mabi.ilp32 \
 march.rv32i_zicsr_zifencei/mabi.ilp32=march.rv32iac_zicsr_zifencei/mabi.ilp32 \
+march.rv32i_zicsr_zifencei/mabi.ilp32=march.rv32iafc_zicsr_zifencei/mabi.ilp32 \
```

Comment:
@tari just to make sure, you want `mabi=ilp32` and not `mabi=ilp32f`, right?

Reply:
Yes, since ilp32f would need to introduce a new library instance.

Reply:
> since ilp32f would need to introduce a new library instance.

If it is foreseeable that `ilp32f` is necessary, we **can** add a new multilib variant.

Note that Zephyr links with `ilp32f` when `CONFIG_FLOAT_HARD=y`, which defaults to `y` when `CONFIG_FPU=y`:

https://github.com/zephyrproject-rtos/zephyr/blob/0f25daa8fe29e694194274c561a9aa6b3b9b7293/arch/riscv/Kconfig#L12-L15

So with the the current patch, you will be forced to set `CONFIG_FLOAT_HARD=n` when building these targets.

Reply:
@tari I will send a follow-up patch to add `rv32if/ilp32f` and map `rv32iafc/ilp32f` to it.